### DISCUSSION TITLE
DCM4CHEE UI WONT CHANGE

### DISCUSSION THREAD

**Discussion by rabzgg** [dcm4che/dcm4chee-arc-light#4451] [Q&A]
![IMG-20240322-WA0011](https://github.com/dcm4che/dcm4chee-arc-light/assets/75366494/ee0e12fc-9fe3-4370-96d3-08893b89e511)
![IMG-20240322-WA0012](https://github.com/dcm4che/dcm4chee-arc-light/assets/75366494/416ffec8-54d8-4dfb-ad9d-65f103163a58)
![IMG-20240322-WA0013](https://github.com/dcm4che/dcm4chee-arc-light/assets/75366494/44db45ff-3f99-42ef-bfae-1f1cc0148682)
![IMG-20240322-WA0014](https://github.com/dcm4che/dcm4chee-arc-light/assets/75366494/4d2c69a7-6cab-4318-909b-e312168dcef1)

I already uploaded a new raw file, but the ui won't change, but I cant find where is the problem. I tried to upload with the wildfly deployments ui and docker copy. What am i missing here?

**Comment by vrindanayak** [comment]
What do you mean by
> but the ui won't change

Your question is not clear - what exactly are you trying to resolve?
